dead_mans_switch_client/data/ir_cron.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="cron_client" model="ir.cron">
+            <field name="name">Dead man&apos;s switch client</field>
```

Comment:
ah, this. not necessary, but allowed https://www.w3.org/TR/REC-xml/#sec-predefined-ent